README.md

```diff
@@ -1,2 +1,23 @@
+
+Just as labs frequently seek to convert backlogged data, they are also often motivated to establish conversion pipelines to automatically convert newly generated data into NWB as they are recorded from acquisition systems. Automatically converting diverse neurophysiology data formats presents challenges due to the wide variety of modalities involved. Even a single lab may employ voltage recording, optical imaging, optogenetics, behavior, and more, each with its own software-dependent formats lacking standardization. A lab-specific conversion pipeline must take all the possible options into account to properly future-proof, or adapt itself to every new experiment that adds or removes a piece of software or hardware. These considerations make conversion pipelines using _pyNWB_ or _MatNWB_ alone difficult to develop and even harder to maintain for experimental researchers with many more immediate concerns vying for their attention.
+
+To mitigate these challenges and promote wider adoption of NWB, we have developed a sophisticated software suite comprising three core tools. The first, _nwbinspector_, is a command-line interface (CLI) tool that ensures the validity of NWB files, enhancing their reliability for uploading to sharing platforms like DANDI. The second, _neuroconv_, is a Python package that facilitates the conversion of neurophysiology data from various proprietary formats to the NWB standard. Additionally, it aids in the development of automatic conversion pipelines, simplifying the process for researchers. The third tool, _NWB GUIDE_, is an interactive and user-friendly application that allows for the conversion of data to the NWB standard without requiring programming expertise. Powered by _neuroconv_, this tool is designed to cater to a broader audience, including researchers with minimal coding experience.
```

Comment:
```suggestion
To mitigate these challenges and promote wider adoption of NWB, we have developed a sophisticated software suite comprising three core tools. The first, _nwbinspector_, is a command-line interface (CLI) tool that ensures the validity of NWB files, enhancing their reliability for uploading to sharing platforms like DANDI. The second, _neuroconv_, is a Python package that facilitates the conversion of neurophysiology data from various proprietary formats to the NWB standard. Its primary purpose is to make it easy to develop automatic conversion pipelines that ingest data from all recording and processing sources used in a neurophysiology experiment. The third tool, _NWB GUIDE_, is an interactive and user-friendly application that allows for the conversion of data to the NWB standard without requiring programming expertise. Powered by _neuroconv_, this tool is designed to cater to a broader audience, including researchers with minimal coding experience.
```

Changing the automated pipelines to be a primary, not additional feature; also, it's emphasizes elsewhere that everything in this section is for simplifying the process users